Apps/Sandcastle/gallery/Model Experimental 3D Models.html

```diff
@@ -36,6 +36,8 @@
         "use strict";
         //Sandcastle_Begin
 
+        Cesium.ExperimentalFeatures.enableModelExperimental = true;
```

Comment:
Since the sandcastle creates `ModelExperimental` objects directly this line isn't needed